.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,7 @@ Fixes #?
 ### Contributor Checklist
 - [ ] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
 - [ ] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
+- [ ] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
```

Comment:
I'm not sure we should link to Apache but I thought it might be a good thing to give people some guidance when they absolutely need to copy something.

Reply:
I do not see how a link could be bad. So 👍 from me.